tools/cross_runner/src/conv.h

```diff
@@ -9,16 +9,31 @@ class Convolution : public DirectMlBaseNode
 {
 public:
     Convolution(const TensorShape& input_shape, const TensorShape& filter_shape, const TensorShape& output_shape,
-        const DML_TENSOR_DATA_TYPE data_type, const dml::TensorPolicy& tensor_policy,
+        const DML_TENSOR_DATA_TYPE data_type, const dml::TensorPolicy& input_tensor_policy, const dml::TensorPolicy& output_tensor_policy,
         const TensorShape& stride_shape, std::uint32_t input_pad, std::uint32_t output_pad,
-            bool use_bias, bool allow_fp16_computations, 
-            IDMLDevice* dml_device, ID3D12Device* d3d12_device, bool disable_mc = false)
+        bool use_bias, bool allow_fp16_computations,
+        IDMLDevice* dml_device, ID3D12Device* d3d12_device, bool disable_mc = false)
         : DirectMlBaseNode(dml_device, d3d12_device)
     {
 
         const dml::TensorDimensions input_dims{ input_shape.n, input_shape.c, input_shape.h, input_shape.w };
         const dml::TensorDimensions filter_dims{ filter_shape.n, filter_shape.c, filter_shape.h, filter_shape.w };
-        const dml::TensorDimensions output_dims{ output_shape.n, output_shape.c, output_shape.h, output_shape.w };
+        dml::TensorDimensions output_dims;
+        if (output_shape.nhwc)
```

Comment:
This doesnt seem right.
Tensor is always in order: NCHW
what determines the layout should be strides calculated object: TensorPolicy output_tensor_policy